sources/packages/web/src/components/common/OfferingFormSubmit.vue

```diff
@@ -60,9 +59,9 @@ export default defineComponent({
     submit: null,
   },
   props: {
-    data: {
-      type: Object as PropType<OfferingFormModel>,
-      default: {} as OfferingFormModel,
+    offeringId: {
+      type: Number,
+      required: true,
```

Comment:
I believe that it should not be required since this component is also used by the `OfferingCreate.vue`. Please take a look, the component should be ready to not receive the ID.
Also please ensure that a smoke test will be executed on offering create/edit/view.

Reply:
I see the following error in OfferingCreate.vue 
![image](https://github.com/bcgov/SIMS/assets/54600590/b4dfc05a-6ae9-402d-8360-ac992f208031)

Reply:
@andrewsignori-aot  I have fixed this and also run the smoke tests for the 3 mentioned scenarios.
@dheepak-aot On reverting back the change in this file, the error mentioned above is also not there.

Reply:
It is showing up when `formMode` prop is set to `required: true` in the `offering-form-submit` component. When the `required` is set to `false`, the error is not shown although `formMode` has a `default` value.

Reply:
@sh16011993 is it clear that the comment above was about the `:data=initialData` property?

Reply:
yes, fixed it @andrewsignori-aot